sermant-plugins/sermant-mq-grayscale/mq-grayscale-rocketmq-plugin/src/main/java/io/sermant/mq/grayscale/rocketmq/utils/RocketMqGrayscaleConfigUtils.java

```diff
@@ -72,7 +76,16 @@ public static String getGrayGroupTag() {
         }
         Optional<GrayTagItem> itemOptional
                 = MqGrayConfigCache.getCacheConfig().getMatchedGrayTagByServiceMeta(MICRO_SERVICE_PROPERTIES);
-        return itemOptional.map(grayTagItem -> standardFormatGroupTag(grayTagItem.getConsumerGroupTag())).orElse("");
+        if (itemOptional.isPresent()) {
+            String consumerGroup = itemOptional.get().getConsumerGroupTag();
+            if (PATTERN.matcher(consumerGroup).matches()) {
+                return itemOptional.get().getConsumerGroupTag();
+            } else {
+                LOGGER.warning(String.format(Locale.ENGLISH, "current consumerGroup tag [%s] not matches pattern "
+                        + "[a-zA-Z0-9_-], modify it and restart service to valid.", consumerGroup));
```

Comment:
"current consumerGroup tag [%s] does not match the required pattern [a-zA-Z0-9_-]. Please modify it to conform to this pattern and restart the service."

Reply:
fixed